v1-scripts/configs/MPCNC/SKR1.3_T8_16T_LCD_32step_DualEndstop

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+incdir=$(dirname $0)/..
+
+use_example_configs CNC/MPCNC
```

Comment:
You don't want this, do you? This is going to have a chicken and egg problem.

Reply:
Or is this just copying the _Bootscreen?

Reply:
This copies the entire config/examples/CNC/MPCNC directory.